### PR TITLE
New README for FastAI

### PR DESCRIPTION
**Pull request by opus111**
New design of FastAI, incorporating code and ideas from the Summer of 2020

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,26 @@
 
 ![Logo](https://raw.githubusercontent.com/opus111/FastAI.jl/master/fastai-julia-logo.png)
 
-This code is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), but differs in implementation in several ways. Most importantly, the original Python code makes heavy use of side-effects where the `Learner` holds different state variables, and other objects access and modify them.
+FluxAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux.  Its goal is enable the development of state-of-the-art models, while freeing the developer from having to implement most of the components.  FastAI allows the design, training and delivery of models that compete with the best in class, using a few lines of code.
```
Comment (darsnack) [inline README.md:8]:
```suggestion
FastAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux. Its goal is to enable creating state-of-the-art models, while freeing the developer from having to implement most of the sub-components. FastAI allows the design, training, and delivery of models that compete with the best in class, using a few lines of code.
```

File: README.md [hunk 2]
```diff
@@ -5,10 +5,26 @@
...
+FluxAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux.  Its goal is enable the development of state-of-the-art models, while freeing the developer from having to implement most of the components.  FastAI allows the design, training and delivery of models that compete with the best in class, using a few lines of code.  
 
-This has been replaced by a more functional design. The state is now transmitted via arguments to `Callbacks` which may then pass them on to `Metrics`.
+FastAI.jl contains thorough documentation, examples and tutorials, but does not contain the source of the core components.  It is an umbrella package combining the functionality specialized packages.  These packages include:
 
-*Note*: this is a package in-development. Expect breaking changes for the foreseeable future, but we want you to test out the package by following the documentation. Any contributions are welcome via PRs/issues.
+- Flux.jl: 100% pure-Julia Deep Learning stack. Provides lightweight abstractions on top of Julia's native GPU and AD support.
```
Comment (darsnack) [inline README.md:12]:
```suggestion
- Flux.jl: 100% pure-Julia Deep Learning stack. Provides lightweight abstractions on top of Julia's core GPU and AD support.
```

File: README.md [hunk 3]
```diff
@@ -5,10 +5,26 @@
...
+
+- FluxTraining.jl: Enables eaily customized training loops, a large library of useful metrics, and many useful utilities
```
Comment (darsnack) [inline README.md:14]:
```suggestion
- FluxTraining.jl: Easily customized training loops, a large library of useful metrics, and many useful utilities (such as logging)
```

File: README.md [hunk 4]
```diff
@@ -5,10 +5,26 @@
...
+
+- FluxONNX.jl: Support for reading and writing models in the ONNX file format.  ONNX enables Transfer Learning by enabling the import of very large pretrained models.
```
Comment (darsnack) [inline README.md:16]: Should we leave this as a comment for now until we have an official ONNX package?

File: README.md [hunk 5]
```diff
@@ -5,10 +5,26 @@
...
+- MLDataSets.jl: A community effort to provide a common interface for accessing common Machine Learning (ML) datasets.
+
+- DataAugmentation.jl: Utilities for augmenting image data
+
+- Metalhead.jl: Computer vision models for Flux 
+
+- Transformers.jl: Implementation of NLP transformer-based models
+
+
+*Note*: this is a package in-development. It is not ready for use, and one should expect major breaking changes for the foreseeable future. However, we are very interested in meeting the desires of the community, so all comments and contributions are welcome via PRs/issues.
```
Comment (darsnack) [inline README.md:29]:
```suggestion
*Note*: this is a package in-development. One should expect major breaking changes for the foreseeable future. But we are very interested in meeting the desires of the community, so all comments and contributions are welcome via PRs/issues.
```

**Review by darsnack — COMMENTED** [review]
I left some suggestions. Maybe we should convert the package names in the list to URL links to the respective GitHub repos?

File: README.md (hunk 4 above)
Comment (darsnack) [reply README.md:16]: Any comments on this?

Comment (lorenzoh) [reply README.md:16]: Yeah, I agree

**Comment by lorenzoh** [comment]
Could you make the packages clickable links to the respective GitHub repositories?

**Review by darsnack — APPROVED** [review]
LGTM

**Review by ToucheSir — APPROVED** [review]

**Pull request merged**